Packs/Lumu/Integrations/Lumu/Lumu.yml

```
@@ -0,0 +1,1031 @@
+    - contextPath: Lumu.RetrieveASpecificIncidentDetails.lastContactDetails.sourceData.DNSPacketExtraInfo.answers.data
+      description: ""
+      type: String
+    - contextPath: Lumu.RetrieveASpecificIncidentDetails.lastContactDetails.sourceData.DNSPacketExtraInfo.opCode
+      description: ""
+      type: String
+    - contextPath: Lumu.RetrieveASpecificIncidentDetails.lastContactDetails.isPlayback
+      description: ""
+      type: Boolean
+    description: |-
+      Get details of a specific Incident.
+
+      | `{incident-uuid}` | uuid of the specific incident |
+      |---|---|
+  - name: lumu-retrieve-a-specific-incident-context
+    arguments:
+    - name: lumu_incident_id
+      description: ""
+    - name: hash
+      description: ""
+    outputs:
+    - contextPath: Lumu.RetrieveASpecificIncidentContext.adversary_id
+      description: ""
+      type: String
+    - contextPath: Lumu.RetrieveASpecificIncidentContext.currently_active
+      description: ""
+      type: Boolean
+    - contextPath: Lumu.RetrieveASpecificIncidentContext.deactivated_on
+      description: ""
+      type: Date
+    - contextPath: Lumu.RetrieveASpecificIncidentContext.mitre.details.tactic
+      description: ""
+      type: String
+    - contextPath: Lumu.RetrieveASpecificIncidentContext.mitre.details.techniques
+      description: ""
+      type: String
+    - contextPath: Lumu.RetrieveASpecificIncidentContext.mitre.matrix
+      description: ""
+      type: String
+    - contextPath: Lumu.RetrieveASpecificIncidentContext.mitre.version
+      description: ""
+      type: String
+    - contextPath: Lumu.RetrieveASpecificIncidentContext.related_files
+      description: ""
+      type: String
+    - contextPath: Lumu.RetrieveASpecificIncidentContext.threat_details
+      description: ""
+      type: String
+    - contextPath: Lumu.RetrieveASpecificIncidentContext.threat_triggers
+      description: ""
+      type: String
+    - contextPath: Lumu.RetrieveASpecificIncidentContext.playbooks
+      description: ""
+      type: String
+    - contextPath: Lumu.RetrieveASpecificIncidentContext.external_resources
+      description: ""
+      type: String
+    - contextPath: Lumu.RetrieveASpecificIncidentContext.timestamp
+      description: ""
+      type: Date
+    description: |-
+      Get details of a specific Incident.
+
+      | `{incident-uuid}` | uuid of the specific incident |
+      |---|---|
+  - name: lumu-comment-a-specific-incident
+    arguments:
+    - name: lumu_incident_id
+      description: ""
+    - name: comment
+      description: ""
+    outputs:
+    - contextPath: Lumu.CommentASpecificIncident.statusCode
+      description: ""
+      type: number
+    description: Get a paginated list of open incidents for the company. The items are listed by the most recent.
+  - name: lumu-retrieve-open-incidents
+    arguments:
+    - name: page
+      description: ""
+    - name: items
+      description: ""
+    - name: adversary-types
```

command parameter should be with underline 
```suggestion
    - name: adversary_types
```

---

suggestion accepted in commit 2bb633df7de2b53b93d8a8cdc1fcaef13e33c6a7